src/app/@ansyn/ol/plugins/annotations/annotations.visualizer.ts

```diff
@@ -193,8 +194,17 @@ export class AnnotationsVisualizer extends EntitiesVisualizer {
 
 	annotationsLayerToEntities(annotationsLayer: FeatureCollection<any>): IVisualizerEntity[] {
 		return annotationsLayer.features.map((feature: Feature<any>): IVisualizerEntity => {
-			const featureJson = { ...feature };
-			delete featureJson.properties.featureJson;
+			const featureJson = cloneDeep(feature);
+			featureJson.properties = {
+				...getInitialAnnotationsFeatureProperties(),
```

Comment:
here should be the decision of taking the style from config and only if it doesn't exist from the hard coded config
you can add the config to production tmpl